client/my-sites/checkout/constants.js

```diff
@@ -0,0 +1 @@
+export const CHECKOUT_US_ADDRESS_FORMAT_COUNTRIES = [ 'AU', 'CA', 'US' ];
```

Comment:
It probably makes more sense to use a whitelist for European style addresses for now, as we don't want to change the address field for all the other countries where we haven't researched whether they actually don't mind US-style format, or where we'd possibly change it for them twice (now to European, then to their country format).